fdbmonitor/fdbmonitor.cpp

```diff
@@ -207,6 +207,7 @@ double timer() {
 #endif
 };
 
+// Produces random integer values i, uniformly distributed on the closed interval [min, max]
```

Comment:
I believe this is true, but just to be clear it's not the `randomInt` function that is used in the code that you modified. In that code, a different function is used which perhaps confusingly is open on the max (i.e. the interval is `[min, max)`).

Reply:
Yeah, I noticed that after I added the comment. I felt the comment can help whoever read the code later, so I kept the comment. :) 

Thanks for pointing this out, anyway!